frontend/src/components/maps/MapFilterBar.tsx

```diff
@@ -35,7 +35,7 @@ const SearchBar: React.FC = () => {
   const {
     values: { searchBy },
     setFieldValue,
-  } = useFormikContext<MapFilterChangeEvent>();
+  } = useFormikContext<IMapFilter>();
```

Comment:
It isn't an event, it's an object with filter parameters.  It is also an interface, so renamed accordingly.

Reply:
Ooh, reminds me I should do the same thing with `SelectOption` and `SelectOptions`